### DISCUSSION TITLE
Erro no Flunt.

### DISCUSSION THREAD

**Discussion by diegomarins33** [balta-io/forum#705] [Tira Dúvidas] [answered]
Desculpem minha ignorância, estou fazendo o curso "Domínios ricos" e na aula "Implementando validações", começamos fazer uso do pacote Flunt para usar notificações. Acontece que no video o código funciona perfeito, mas no meu Rider, fica com erro em "Notifiable" (na hora de herdar), ele pede pra por o "<>", mas ai se eu ponho, o erro (grifado vermelho) vai pra cima do <>, como posso resolver isso? 

```
using Flunt.Notifications;

namespace PaymentContext.Shared.Entities
{
    public abstract class Entity : Notifiable<>
    {
        public Entity()
        {
            Id = Guid.NewGuid();
        }

        public Guid Id { get; private set; }
    }
}
```

**Comment by diegomarins33** [comment]
![image](https://github.com/user-attachments/assets/92a37d65-72b4-4780-a297-93e0d3a2ddf1)

![image](https://github.com/user-attachments/assets/a5924d24-6ed4-4c25-aafa-69212e6d5242)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado:
https://github.com/balta-io/forum/discussions/120
